data/json/items/gun/8x40mm.json

```diff
@@ -204,4 +204,236 @@
     "magazine_well" : 2,
     "flags": [ "WATERPROOF_GUN", "NEVER_JAMS" ]
   }
+[
```

Comment:
An odd separator here is breaking the .json parser. Also, what's with all the duplicated guns listed in here?